cob_bringup/components/cam3d_visionary_t.launch

```diff
@@ -13,5 +14,13 @@
 		<arg name="yaml_name" value="$(arg name)"/>
 		<arg name="sim" value="$(arg sim)"/>
 	</include>
-
+	<include if="$(arg flip)" file="$(find cob_bringup)/drivers/image_flip.launch">
+		<arg name="robot" value="$(arg robot)"/>
+		<arg name="camera_name" value="$(arg name)"/>
+		<arg name="pointcloud_in" value="/$(arg name)/depth/points"/>
+		<arg name="pointcloud_out" value="/$(arg name)_upright/depth/points"/>
+		<!-- Sick3DCS does not have color image, and image flip cannot process images that are not CV_8UC3 -->
```

Comment:
@ipa-rmb FYI
I don't know whether this is intended...

Reply:
I do not care about that camera at the moment. Image flip would need to be extended to handle other types of images as well, the current code fpr 8UC3 images is speed optimized and not general. We can later add a slower general fall back solution for other image types if anyone needs that at all.

Reply:
I created a new issue for this: https://github.com/ipa320/cob_perception_common/issues/85